config/routes.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
+
+# rubocop:disable Metrics/BlockLength
```

Comment:
Entirely related and unrelated to this PR (aka the pattern already existed) With Rails routing I would expect to see these routes within the resource.  Something more like
```
  resources :works do 
    member do
      ...
      put "approve"
      put "withdraw"
      ....
      put "assign-curator"
      get "datacite"
    end
  end
```
Then the rubocop disable would not be needed

Reply:
I updated the route to use PUT instead of POST. 

Let's open a separate issue/PR if we want to refactor the routes and/or pass the parameters in the body. Keep in mind that a lot of these routes will not stay for long, they are pretty exploratory at this point.

Reply:
https://github.com/pulibrary/pdc_describe/issues/212